deploy/delete_operators.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+PATH_TO_KUBECONFIG=${HOME}/.kube/config_eks
```

Comment:
Let's move this logic into Makefile.
Also PATH_TO_KUBECONFIG should be configurable with default value.